examples/permissioned-token/scripts/admin/cease.js

```diff
@@ -0,0 +1,99 @@
+const {
+  balanceOf, executeTransaction
+} = require('@algo-builder/algob');
+const { types } = require('@algo-builder/runtime');
```

Comment:
note: types import is changed to `require(@algo-builder/web)` from `require(@algo-builder/runtime)`